src/scripts/main.js

```diff
@@ -358,3 +358,44 @@ const people = [
 console.log(people); // you can remove it
 
 // write your code here
+
+const table = document.querySelector('.dashboard');
```

Comment:
Ensure that the element with the class 'dashboard' exists in the DOM before attempting to append rows to it. If it doesn't exist, this will cause an error.